### PR TITLE
Updating to db-extractor-common 9

### PR DESCRIPTION
**Pull request by pivnicek**
will fix: #28 and #33

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,5 +29,23 @@
         "psr-4": {
             "Keboola\\DbExtractor\\Tests\\": "tests/Keboola/DbExtractor"
         }
+    },
+    "scripts": {
+        "tests": "./wait-for-it.sh mysql:3306 -t 30 -- ./vendor/bin/phpunit",
+        "phpstan": "phpstan analyse --no-progress --level=max ./src ./tests",
+        "phpcs": "phpcs -n --ignore=vendor --extensions=php .",
+        "phpcbf": "phpcbf -n --ignore=vendor --extensions=php .",
+        "build": [
```
Comment (tomasfejfar) [inline composer.json]: same as with mssql (phplint, waitforit in docker compose...

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 2]
```diff
@@ -7,34 +7,32 @@
...
+use PDOException;
 
 class MySQL extends Extractor
 {
-    const TYPE_AUTO_INCREMENT = 'autoIncrement';
-    const TYPE_TIMESTAMP = 'timestamp';
+    public const TYPE_AUTO_INCREMENT = 'autoIncrement';
+    public const TYPE_TIMESTAMP = 'timestamp';
 
+    /** @var  string -- database name from connection parameters */
```
Comment (tomasfejfar) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: what does the `--` do?

File: tests/Keboola/DbExtractor/MySQLSSLTest.php [hunk 3]
```diff
@@ -26,7 +26,7 @@ public function testCredentials()
             'ca' => file_get_contents($this->dataDir . '/mysql/ssl/ca.pem'),
             'cert' => file_get_contents($this->dataDir . '/mysql/ssl/client-cert.pem'),
             'key' => file_get_contents($this->dataDir . '/mysql/ssl/client-key.pem'),
-//			'cipher' => '',
```
Comment (tomasfejfar) [inline tests/Keboola/DbExtractor/MySQLSSLTest.php]: is there a time when you need to uncomment this? And is that regularly? If not, you can delete it.

File: src/Keboola/DbExtractor/Extractor/MySQL.php (hunk 2 above)
Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: nothing, just a delimiter to indicate comment

File: tests/Keboola/DbExtractor/MySQLSSLTest.php (hunk 3 above)
Comment (pivnicek) [reply tests/Keboola/DbExtractor/MySQLSSLTest.php]: yes, that's been there forever, I'll get rid of it 👍

File: composer.json (hunk 1 above)
Comment (pivnicek) [reply composer.json]: yeah, in follow-up as you say.  I'll make an issue for it. 👍

**Pull request merged**